test/integration/ci-operator-configresolver.sh

```diff
@@ -1,65 +1,88 @@
 #!/bin/bash
 source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
 
-rsync_tmp_dir=$(mktemp -d)
 function cleanup() {
-    rm -rf $rsync_tmp_dir
     os::test::junit::reconcile_output
     os::integration::configresolver::check_log
     os::cleanup::processes
 }
 trap "cleanup" EXIT
 
 suite_dir="${OS_ROOT}/test/integration/ci-operator-configresolver/"
-cp -a "${suite_dir}"/* "${BASETMPDIR}"
-cp -a "${OS_ROOT}/test/multistage-registry" "${BASETMPDIR}"
-actual="${BASETMPDIR}/actual"
-mkdir -p "${actual}"
-expected="${suite_dir}/expected"
+unused_dir=${suite_dir}/unused
 
 os::test::junit::declare_suite_start "integration/ci-operator-configresolver"
-# This test validates the ci-operator-configresolver tool
 
-os::integration::configresolver::start "${BASETMPDIR}/configs" "${BASETMPDIR}/multistage-registry/registry" "${BASETMPDIR}/config.yaml"
+test_validation() {
+    local out=${BASETMPDIR}/validation
+    local cmd=(
+        ci-operator-configresolver
+        --config "${unused_dir}/"
+        --jobs "${unused_dir}/"
+        --prow-config "${unused_dir}/unused.md"
+        --validate-only)
+    # Validate empty registry directory.
+    os::cmd::expect_success "${cmd[*]} --registry ${unused_dir}/"
+    # Unresolved parameters are valid at this stage.
+    os::cmd::expect_success \
```

Comment:
We don't care about line width

Reply:
I do, but more importantly, if the comment applies to all commands broken over multiple lines in this file: I find it much harder to read long lines with several flags/arguments.

(but I guess I'll have to live with it)

Reply:
Can we expense you a 4k monitor? :)